var/spack/repos/builtin/packages/r-cluster/package.py

```diff
@@ -34,6 +34,6 @@ class RCluster(RPackage):
     url      = "https://cran.rstudio.com/src/contrib/cluster_2.0.6.tar.gz"
     list_url = "https://cran.r-project.org/src/contrib/Archive/cluster"
 
-    version('2.0.6', 'e7d0660b05c65f4d886ebf7f046939ab')
+    version('2.0.6', 'ce82c6c8024043943c567b821efe3fee')
```

Comment:
Can you ask the developers why the hash changed? It might just be that they found a bug and re-released. Or they could have been hacked and Spack would be distributing a virus...

Reply:
The email was sent. Waiting for their response now.

Reply:
I got their reply as follows:

I have asked the CRAN team -- CRAN is the source of all CRAN
packages, and "spack" is not known to me or them and is hence
irrelevant to us and to the R community at large, AFAIK.
[[I'm still slightly curious what it is.. ]]

The CRAN team confirms they have not changed cluster_2.0.6 at all.
As I told you below, 2.0.6. is now outdated by 2.0.7 in any
case.

Reply:
That's pretty weird... If we don't know why the hash changed, I would prefer to remove that version from the package and add the new version instead.